src/components/CodeSearchResults/CodeSearchResults.scss

```diff
@@ -3,7 +3,6 @@
 .code-search-results {
   .loading {
     @extend .mb-3;
-    height: 220px;
```

Comment:
Note: Setting `height` here was causing the table loading overlay box to take up too much space. The height on the initial load of the table data is already defined in the table component styles anyways :)